sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_models.py

```diff
@@ -54,24 +59,33 @@ def __init__(self, key_id, algorithm, signature):
 class VerifyResult:
     """The result of a verify operation.
 
-    :param bool result: Whether the signature is valid
+    :param str key_id: The encryption key's Key Vault identifier
+    :param bool is_valid: Whether the signature is valid
+    :param algorithm: The encryption algorithm used
+    :type algorithm: ~azure.keyvault.keys.crypto.EncryptionAlgorithm
     """
 
-    def __init__(self, result):
-        self.result = result
-
+    def __init__(self, key_id, is_valid, algorithm):
+        self.key_id = key_id
+        self.is_valid = is_valid
+        self.algorithm = algorithm
 
-class UnwrapKeyResult:
+class UnwrapResult:
     """The result of an unwrap key operation.
 
-    :param bytes unwrapped_bytes: The unwrapped key's bytes
+    :param str key_id: The signing key's Key Vault identifier
+    :param algorithm: The signature algorithm used
+    :type algorithm: ~azure.keyvault.keys.crypto.SignatureAlgorithm
+    :param bytes key: The unwrapped key's bytes
```

Comment:
I think "The unwrapped key" is sufficient.

Reply:
switched, thanks!